cmd/neofs-cli/modules/container/set_eacl.go

```diff
@@ -36,6 +41,19 @@ Container ID in EACL table will be substituted with ID from the CLI.`,
 		pk := key.GetOrGenerate(cmd)
 		cli := internalclient.GetSDKClientByFlag(cmd, pk, commonflags.RPC)
 
+		if flagVarsSetEACL.preCheck {
```

Comment:
It is hard to know about the problem, even having the description of this flag.
What about replacing it with `--no-precheck`? This way user will be satisfied by default and infra team can provide the flag to speed up things.
What do you think? @aprasolova @carpawell @realloc @532910

Reply:
Good idea. Checking by default and having a flag to change this behaviour looks logical.

Reply:
agree

Reply:
Inverted the flag.